baselines/teams.md

```diff
@@ -88,114 +87,141 @@ need.
 - N/A
 
 ### Implementation
-
+All the settings in this section are configured in the [**Microsoft Teams admin
+    center**](https://admin.teams.microsoft.com).
+#### MS.TEAMS.1.1v1, instructions:
```

Comment:
Let me know what you all decide on this. No preference from me.
If we change it, we need to update the [markdown style guide](https://github.com/cisagov/ScubaGear/blob/Baseline-Markdown-Style-Guide/baselines/Example.md) and at a minimum AAD and SharepointOneDrive because I'm using the same format as this Teams pull request branch.

Reply:
Based on latest style guide changes, removed the comma in all instructions sub-headings.